docs/architecture/self-hosted/self-hosted.md

```diff
@@ -21,6 +21,10 @@ There are a couple of ways to install the self-managed version of Prophecy:
 
 Once Prophecy is installed, you'll have to manually perform upgrades, backups, restores, etc.
 
+:::info
+For personalized help, contact [support](https://prophecy.zendesk.com/). If applicable, send along your [logs](download-logs.md) to help us troubleshoot.
+:::
+
```

Comment:
Please add in this note that the "Download Logs" feature is enabled by default on Prophecy v3.4.1.0+

Reply:
will do

Reply:
I'll add it to the download logs page, assuming they have to go there anyway to learn about how to use it

Reply:
@kathweinschenkprophecy I like the mention of 3.4.1.0+, however I'm not seeing **by default**

Maybe we should remove the steps to enable the feature. I'll ping in #customer-success channel to ask on this.